docs/demo-plugin.js

```
@@ -0,0 +1,288 @@
+  function renderReactDemo(code, demoId) {
+    const rootDiv = document.getElementById(`${demoId}-root`);
+    if (!rootDiv || typeof React === 'undefined' || typeof ReactDOM === 'undefined') {
+      rootDiv.innerHTML = '<div class="demo-notice">React not loaded. Please include React and ReactDOM in your page.</div>';
+      return;
+    }
+
+    try {
+      // Transform JSX if needed (requires Babel standalone)
+      if (typeof Babel !== 'undefined') {
+        const transformed = Babel.transform(code, { presets: ['react'] }).code;
+        eval(transformed);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)